src/styles/rhd-theme/components/_buttons.scss

```diff
@@ -1,5 +1,5 @@
 
-@import "../@patternfly/patternfly/components/Button/button.scss";
```

Comment:
Imports should be treated as though the `styles` directory is a sibling within the `node_modules` so this should really just be `../../@patternfly...` since it is two directories down

Reply:
Should be all set now.